extra/1-radio-stations.js

```diff
@@ -29,7 +33,13 @@ function getAllFrequencies() {
  */
 // `getStations` goes here
 function getStations () {
-  
+ let radioStFreq =[]; 
+let activeFrequency  = getAllFrequencies();
+for (i=0; i<=activeFrequency.length; i++);
+  if (isRadioStation(activeFrequency[i]=== true)) {
```

Comment:
The second set of round brackets should be closed after activeFrequency[i], before the strict equality operator.